tests/protos/kitchen-sink.proto

```diff
@@ -7,6 +7,7 @@
 package some_package;
 
 import "tests/protos/options.proto";
+import "google/protobuf/descriptor.proto";
```

Comment:
imports in pbnj are pretty broken right now. they were totally incompatible with how protoc resolves imports, and we turned them off.

This works in protoc because in protoc, it's standard to do `--proto_path=.:/usr/local/include`